### PR TITLE
Issue 8: null file generated on empty editor

### PR DESCRIPTION
**Pull request by gtico80**
Issue #8: disabled download button

### CODE REVIEW

**Comment by libremente** [comment]
Looks good to me! :+1:

File: src/app/components/sidebar.js [hunk 1]
```diff
@@ -105,6 +105,9 @@ export default class sidebar extends Component {
 
   download(data) {
     //has dom
+	if (!data || data.length ==0){
```
Comment (libremente) [inline src/app/components/sidebar.js]:
```suggestion
	if (!data || data.length == 0){
```

File: src/app/components/sidebar.js [hunk 2]
```diff
@@ -249,9 +252,9 @@ export default class sidebar extends Component {
             </a>
           </div>
           <div className="sidebar__footer_item">
-            <a href="#">
+            <a href="#" className={!yaml? 'disabled' : 'enabled'}>
```
Comment (libremente) [inline src/app/components/sidebar.js]:
```suggestion
            <a href="#" className={!yaml ? 'disabled' : 'enabled'}>
```

File: src/app/components/sidebar.js [hunk 3]
```diff
@@ -249,9 +252,9 @@ export default class sidebar extends Component {
...
               <img src={img_download} alt="dowload" />
-              <span className="action" onClick={() => this.download(yaml)}>
+              <span className="action" onClick={!yaml? null : () => this.download(yaml)}>
```
Comment (libremente) [inline src/app/components/sidebar.js]:
```suggestion
              <span className="action" onClick={!yaml ? null : () => this.download(yaml)}>
```

**Comment by gtico80** [comment]
Suggestions accepted

**Comment by sebbalex** [comment]
solution tested and it works as expected. 👍

**Comment by libremente** [comment]
Great @gtico80, thanks for your contribution! Merging now :+1:

**Pull request merged**